modules/virt-configuring-vm-real-time.adoc

```diff
@@ -49,56 +50,144 @@ spec:
         memory:
           guest: 4Gi
           hugepages:
-            pageSize: 1Gi <6>
+            pageSize: 1Gi # <6>
         terminationGracePeriodSeconds: 0
 # ...
 ----
 <1> This annotation specifies that load balancing is disabled for CPUs that are used by the container.
 <2> This annotation specifies that the CPU quota is disabled for CPUs that are used by the container.
 <3> This annotation specifies that interrupt request (IRQ) load balancing is disabled for CPUs that are used by the container.
-<4> The number of sockets inside the VM. 
+<4> The number of sockets inside the VM.
 <5> The number of cores inside the VM. This must be a value greater than or equal to `1`.
 <6> The size of the huge pages. The possible values for x86-64 architectures are `1Gi` and `2Mi`. In this example, the request is for 4 huge pages of size 1 Gi.
-
 . Apply the `VirtualMachine` manifest:
 +
 [source,terminal]
 ----
 $ oc apply -f <file_name>.yaml
 ----
-
-. Configure the guest operating system. The following example shows the configuration steps for a {op-system-base} 8 operating system:
+. Configure the guest operating system. The following example shows the configuration steps for a {op-system-base} 9 operating system:
 .. Run the following command to connect to the VM console:
 +
 [source,terminal]
 ----
 $ virtctl console <vm_name>
 ----
-
-.. Configure huge pages by using the GRUB boot loader command-line interface. In the following example, 8 1G huge pages are specified.
++
+[NOTE]
```

Comment:
If the suggested changes on lines 70 and 74 are acceptable, then we can remove this note.

Reply:
This note is required.
Is the phrasing correct?

Reply:
The note placement seems weird. Is it possible to add an actual step instead instead of an admonition? Something like:
`.. Log in to the guest as a root user by using the following command:`

Reply:
There are at least thee routes a user could take:
1. Login directly as root (this is somethings disabled).
2. Login as a regular user and switching to root (using `su` or `sudo su`).
3. Login as a regular user and executing each command using `sudo`.

This is why I think we should not limit the instructions to use a single command.
WDYT?

Reply:
For the purpose of this procedure, wouldn't option 2 make the most sense? Especially because all the commands after `virtctl console <vm_name>` have the `#` prompt which indicates a root user. How about `.. If you are not already logged in as a root user, switch to the root user account to execute the remaining configuration steps.`

I am not completely opposed to a Note, but we would need to change the indentation to left so that it doesn't seem like it applies to the previous step (visually). Adjusting the indentation might cause the list numbering to break.

Reply:
Done.